File: Problems/ICPC Asyut Sheets/Sheet #1 (Data type - Conditions)/S. Interval.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    float x;
    cin >> x;
    if (x > 100 || x < 0)
        cout << "Out of Intervals";
    else
    {
        cout << "Interval ";
        if (x >= 0 && x <= 25)
            cout << "[0,25]";
        else if (x > 25 && x <= 50)
            cout << "(25,50]";
        else if (x > 50 && x <= 75)
            cout << "(50,75]";
        else if (x > 75 && x <= 100)
            cout << "(75,100]";
    }

    return 0;
}

File: Problems/ICPC Asyut Sheets/Sheet #1 (Data type - Conditions)/S. Interval.py
x = float(input())

if (x > 100 or x < 0): 
    print("Out of Intervals")

else:
    print("Interval",end=" ")
    if x >= 0 and x <= 25: print("[0,25]")
    elif x > 25 and x <= 50: print("(25,50]")
    elif x > 50 and x <= 75: print("(50,75]")
    elif x > 75 and x <= 100: print("(75,100]")